analysis/descriptives/venn_diagram.R

```diff
@@ -13,12 +13,15 @@
 ##           reporting from different data sources
 ## Output:   Venn diagrams in SVG files, venn_diagram_number_check.csv
 ## =============================================================================
+library(data.table)
+library(readr)
+library(dplyr)
 
 args <- commandArgs(trailingOnly=TRUE)
 
 if(length(args)==0){
   # use for interactive testing
-  population <- "electively_unvaccinated"
+  population <- "vaccinated"
```

Comment:
Use of `population` here, see previous comment.

Reply:
This has been updated as per your suggestion

Reply:
Thank you!